boards/common/esp32/Makefile.include

```diff
@@ -8,5 +8,11 @@ PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.SLAB_USBtoUART*)))
 include $(RIOTMAKE)/tools/serial.inc.mk
 
 # reset tool configuration
-RESET ?= esptool.py
-RESET_FLAGS ?= --port $(PORT) --before default_reset run
+ifeq (1,$(ESP_RESET_DTRTOGGLE))
```

Comment:
This change and the distinction is not needed any longer. With PR #12763, the reset tool  `dist/tools/esptool/espreset.py` was added and the `RESET` variable in `boards/common/esp32/Makefile.include` is set accordingly.